packages/gestalt/src/BannerOverlay.js

```diff
@@ -0,0 +1,276 @@
+// @flow strict
+import { Children, type Element, type ElementConfig, type Node as ReactNode } from 'react';
+import Avatar from './Avatar';
+import styles from './BannerOverlay.css';
+import PrimaryAction from './BannerOverlay/PrimaryAction';
+import {
+  BannerOverlayAvatarThumbnail,
+  BannerOverlayIconThumbnail,
+  BannerOverlayImageThumbnail,
+  BannerOverlayMessage,
+} from './BannerOverlay/subcomponents';
+import Box from './Box';
+import Button from './Button';
+import ButtonLink from './ButtonLink';
+import { useColorScheme } from './contexts/ColorSchemeProvider';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider';
+import { useDeviceType } from './contexts/DeviceTypeProvider';
+import Flex from './Flex';
+import Icon from './Icon';
+import IconButton from './IconButton';
+import Image from './Image';
+import Layer from './Layer';
+import Link from './Link';
+import Text from './Text';
+import { type Indexable } from './zIndex';
+
+const DEFAULT_COLORS = {
+  lightModeBackground: 'default',
+  darkModeBackground: 'elevationRaised',
+  textColor: 'default',
+  iconColor: 'white',
+};
+
+type Props = {
+  /**
+   * Adds a dismiss button to BannerOverlay. See the [Dismissible variant](https://gestalt.pinterest.systems/web/banneroverlay#Dismissible) for more info.
+   * The `accessibilityLabel` should follow the [Accessibility guidelines](https://gestalt.pinterest.systems/web/banneroverlay#Accessibility).
+   *
+   */
+  dismissButton?: {
+    accessibilityLabel?: string,
```

Comment:
accessibilityLabel?: string >> Can we just remove this one and use a default string from provider?  "Dismiss banner"

Reply:
i think we can just do onDismiss

Reply:
DONE